docs/vocs/docs/pages/book/advanced-usage/sdk.mdx

```diff
@@ -160,3 +183,35 @@ The halo2 aggregation proving key `agg_halo2.pk` used in the above example is la
 :::
 
 Note that `DEFAULT_PARAMS_DIR` is the directory where Halo2 parameters are stored by the `cargo openvm setup --evm` CLI command. For more information on the setup process, see the `EVM Level` section of the [verify](/book/writing-apps/verifying-proofs) doc.
+
+## Another Example: Using the Standard SDK
+
+For many programs with modular arithmetic and/or cryptographic operations, the RISC-V instruction set alone may not be enough to achieve the performance desired. OpenVM supports many extensions to accelerate said operations, a standard set of which are included inside the standard SDK configuration.
+
+Take, for example, this toy program that checks if several guest library implementations of [`IntMod`](/book/acceleration-using-extensions/algebra#available-traits-and-methods) are fields by loosely checking if their moduli are prime.
+
+```rust
+// [!include ~/snippets/examples_sdk_guest/little/src/main.rs]
+```
+
+Its `openvm.toml`, as with `Sdk::riscv32()` above, must follow a [specific set configuration](/book/advanced-usage/sdk#standard-sdk-configuration-and-supported-extensions).
+
+Simply initialize `Sdk` using `Sdk::standard()` and build, prove, and verify as above.
+
+```rust
+// [!include ~/snippets/examples_sdk/sdk_app.rs:init]
+```
+
+### Standard SDK Configuration and Supported Extensions
```

Comment:
can you move this earlier to be together with the Present Configurations section? it feels more natural there since it's more important than just an example